src/usr/local/www/system_certmanager.php

```diff
@@ -195,6 +195,20 @@
 }
 
 if ($_POST['save']) {
+	// If passwords are given, check that they match and verify their length
+	function validatePasswords(& $input_errors, $pw1, $pw2) {
```

Comment:
Why not put this function somewhere common (like `certs.inc`, or `guiconfig.inc`) and use it here and on `system_camanager.php`? It would need a more specific name, though, such as `validate_cert_passwords()`.

Reply:
That's not a bad idea considering that it'll be needed for the OpenVPN wizard too.